bff/src/Altinn.Authentication.UI/Altinn.Authentication.UI.Integration/UserProfiles/AccessManagementClient.cs

```diff
@@ -92,7 +97,66 @@ public async Task<PartyExternal> GetParty(int partyId)
         }
         catch (Exception ex)
         {
-            _logger.LogError(ex, "Authentication.UI // PartyClient // GetPartyFromReporteeListIfExists // Exception");
+            _logger.LogError(ex, "Authentication.UI // AccessManagementClient // GetPartyFromReporteeListIfExists // Exception");
+            throw;
+        }
+    }
+
+    /// <inheritdoc />
+    public async Task<List<DelegationResponseData>?> CheckDelegationAccess(string reporteePartyId, List<Right> request)
```

Comment:
Er dette partyid eller er det organisasjonsnr?  Hvis det er organisasjonsnr ikke bruk partyid som begrep

Reply:
the endpoint allows for both, but I think they expect partyId as default, and that is what we use